superset/Dockerfile

```diff
@@ -83,7 +83,8 @@ USER stackable
 WORKDIR ${HOME}
 
 CMD ["/bin/sh", "-c", \
-    "gunicorn \
+    "superset init && \
```

Comment:
This is a possible solution. Another (more elegant) solution would be to add this to the pod command or create an init container.